src/main/webapp/index-script.js

```diff
@@ -1,16 +1,74 @@
+var map;
```

Comment:
We did the same thing here except I made it a const, and you made it a var. What is your reasoning for choosing var? 

Personally, I made it a const because it was already a const in the createMap() method, but I'm open to changing it to var - just would like to know why!